packages/Combobox/src/combobox.scss

```diff
@@ -131,6 +131,11 @@
     &--active,
     &.active {
         border-color: var(--ids-combobox-border-color-active);
+
+        &:focus,
```

Comment:
I don't think this is supposed to be inside --active class

Reply:
Opening a Select/Combobox gives the select both the active and focus class. This is why this is inside active.